### ISSUE TITLE
Change logo display markup in the footer

### ISSUE THREAD

**Issue by AyanChoudhary** [creativecommons/cccatalog-frontend#1117]
## Problem

Change logo display markup in the footer. 

## Description

Currently we use an svg with a ```<use>``` tag to display the logo, but this results in a reduced accessibility score as that element does not have a useful aria-label. The common practices of setting ```aria-role="img"``` and providing a title also don't work in this scenario.

Also it is not recommended to use svg with ```<use>``` for better accessibility purposes [Accessible SVG](https://css-tricks.com/accessible-svgs/#2-inline-svg)

## Alternatives

We might convert the logo to use with an ```<img>``` tag or use a standardized svg format.

## Additional context

Lighthouse Report
```
Links do not have a discernible name
Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users.
Failing Elements
--
a<a class="main-logo" href="https://creativecommons.org" target="_blank"><div class="has-text-white"><svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid meet" viewBox="0 0 304 73"> <use href="#logomark"></use> </svg></div></a>
```

## Implementation

- [x] I would be interested in implementing this feature.

**Comment by Aakash2408** [comment]
Hey @AyanChoudhary  Can I take this?

**Comment by zackkrida** [comment]
@Aakash2408 please do!

**Comment by zackkrida** [comment]
Because this logo isn't significant (we don't _need_ screen reader users to know it exists), we can use the following fixes:

- [ ] `focusable="false"` on the svg tag
- [ ] `aria-hidden="true"` on the use tag

**Comment by neeraj-2** [comment]
I tried the same as @zackkrida said:I got this:the first screenshot is showing the performance before change and second screenshot is having increased performance as per lighthouse report ,i just changed 
1.focusable="false" on the svg tag and 
 2.aria-hidden="true" on the use tag
![lighthouse_report_before_change](https://user-images.githubusercontent.com/56186425/92305698-31110f80-efa7-11ea-98a6-c59bd2c8342c.PNG)
![lighthousereport_after_change](https://user-images.githubusercontent.com/56186425/92305703-40905880-efa7-11ea-95b2-27259cc8eaef.PNG)
![changes](https://user-images.githubusercontent.com/56186425/92305708-4ede7480-efa7-11ea-9944-efa2be889b14.PNG)

**Comment by neeraj-2** [comment]
so,is it the same what we required in this issue or something else,please let me know!
